pkg/antctl/raw/multicluster/create/member_token.go

```diff
@@ -83,10 +83,8 @@ func memberTokenRunE(cmd *cobra.Command, args []string) error {
 	createdRes := []map[string]interface{}{}
 	defer func() {
 		if createErr != nil {
-			fmt.Fprintf(cmd.OutOrStderr(), "Failed to create new member token. Deleting the created resources\n")
-			if err := common.Rollback(cmd, memberTokenOpts.k8sClient, createdRes); err != nil {
-				fmt.Fprintf(cmd.OutOrStdout(), "Failed to rollback: %v\n", err)
-			}
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to create the member token. Deleting the created resources\n")
```

Comment:
According to the pattern in `pkg/antctl/raw/multicluster/common/common.go`, error should go to `ErrOrStderr`? Or I misunderstand the pattern?

Reply:
I think this is not an error message, but just a notice about start deleting created resources.

Errors are printed earlier already.

Reply:
got it